src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -115,13 +91,10 @@ func hasCustomChainedServer(configPath, name string) bool {
 	// The config will have more than one but fewer than 10 chained servers
 	// if it has been given a custom config with a custom chained server
 	// list
-	return nc > 0 && nc < 10
-}
-
-func isGoodConfig(configPath string) bool {
-	log.Debugf("Checking config path: %v", configPath)
-	fi, exists := exists(configPath)
-	return exists && hasCustomChainedServer(configPath, fi.Name())
+	if nc <= 0 || nc > 10 {
```

Comment:
Actually the only thing we've validating is that is has custom chained servers.  Still, at some point it may expand, so I've renamed it.